src/routes/users.js

```diff
@@ -25,8 +25,11 @@ router.get('/:id', ensureLoggedIn(), catchAsync(async (req, res) => {
 // View all users
 router.get('/', ensureLoggedIn(), catchAsync(async (req, res) => {
   const users = await User.findAll({});
-
-  res.render('users/index', {users});
+  if(req.user.role == 'Borrower') {
```

Comment:
This check should be across _all_ user-related routes that are only for admins, e.g. updating a user. It might be good to make it a re-usable function, like `ensureLoggedIn()`, (e.g. `ensureAdmin`).